rskj-core/src/test/java/co/rsk/peg/BridgeSerializationUtilsTest.java

```diff
@@ -1123,26 +1124,23 @@ private void testSerializeAndDeserializeFederation(
                 members.add(new FederationMember(new BtcECKey(), new ECKey(), new ECKey()));
             }
 
-            Federation testFederation = FederationFactory.buildStandardMultiSigFederation(
-                members,
-                Instant.now(),
-                123,
-                bridgeConstants.getBtcParams()
-            );
+            FederationArgs federationArgs = new FederationArgs(members, Instant.now(), 123,
```

Comment:
Same as above, i don't think its worth being a class instance value.
It's only used as a creationBlockNumber in two places in a 1200-line file.
but i agree it could be placed inside a named variable